src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1693,9 +1693,14 @@ static void addIfNotNull(ArrayBuilder<Cci.ICustomAttribute> builder, Synthesized
             return new Cci.TypeReferenceWithAttributes(typeRef, builder.ToImmutableAndFree());
         }
 
-        internal static bool IsWellKnownTypeInAttribute(this TypeSymbol typeSymbol) => typeSymbol.IsWellKnownInteropServicesTopLevelType("InAttribute");
+        internal static bool IsWellKnownTypeInAttribute(this TypeSymbol typeSymbol)
+            => typeSymbol.IsWellKnownInteropServicesTopLevelType("InAttribute");
 
-        internal static bool IsWellKnownTypeUnmanagedType(this TypeSymbol typeSymbol) => typeSymbol.IsWellKnownInteropServicesTopLevelType("UnmanagedType");
+        internal static bool IsWellKnownTypeUnmanagedType(this TypeSymbol typeSymbol)
+            => typeSymbol.IsWellKnownInteropServicesTopLevelType("UnmanagedType");
+
+        internal static bool IsWellKnownTypeIsInitOnlyType(this TypeSymbol typeSymbol)
```

Comment:
>Type [](start = 54, length = 4)

It looks like naming is not consistent with helpers above due to the "Type" suffix.